### PR TITLE
[EdgeTPU] Refactoring createCfg function in OneExplorer.ts

### PR DESCRIPTION
**Pull request by Hyeon-Uk**
This PR is a refactoring of the createCfg function for future scalability considerations.

Refactoring the cfg information required for the createCfg function so that it can be received and used by the corresponding cfg Data

### CODE REVIEW

**Comment by SeungHyunH** [comment]
LGTM

**Comment by imsw0529** [comment]
LGTM

File: media/EdgetpuCfgEditor/cfgeditor.css [hunk 1]
```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (profornnan) [inline media/EdgetpuCfgEditor/cfgeditor.css]: It seems like you need to change the copyright year for the newly created files. 😢 

```suggestion
 * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
```

**Review by profornnan — CHANGES_REQUESTED** [review]

File: media/EdgetpuCfgEditor/cfgeditor.css (hunk 1 above)
Comment (Hyeon-Uk) [reply media/EdgetpuCfgEditor/cfgeditor.css]: i fixed it. thanks :)

**Review by profornnan — APPROVED** [review]

**Pull request closed without merging**